core/build.gradle.kts

```diff
@@ -157,6 +157,8 @@ tasks.withType<KorroTask> {
 // This task installs the pre-commit hook to the local machine the first time the project is built
 // The pre-commit hook contains the command to run processKDocsMain before each commit
 val installGitPreCommitHook by tasks.creating(Copy::class) {
+    doNotTrackState(/* reasonNotToTrackState = */ "Fails on TeamCity otherwise.")
```

Comment:
curious

Reply:
I think it's because TC has a protected/non-existent .git folder